Makefile

```diff
@@ -48,7 +48,7 @@ EXTRACT_XISO = $(NXDK_DIR)/tools/extract-xiso/extract-xiso
 TOOLS        = cxbe vp20compiler fp20compiler extract-xiso
 NXDK_CFLAGS  = -target i386-pc-win32 -march=pentium3 \
                -ffreestanding -nostdlib -fno-builtin -fno-exceptions \
-               -I$(NXDK_DIR)/lib -I$(NXDK_DIR)/lib/xboxrt \
+               -I$(NXDK_DIR)/lib -I$(NXDK_DIR)/lib/xboxrt/libc_enhancements \
```

Comment:
nit: Is it just me, or would `libc_extensions` be more suitable (or is that term confusing / used already by libc)?

Reply:
Agreed and changed.